onecodex/distance.py

```diff
@@ -21,28 +22,20 @@ def alpha_diversity(self, metric="simpson", rank="auto"):
         """
         import skbio.diversity
 
-        if metric not in ("simpson", "chao1", "shannon"):
+        if metric not in AlphaDiversityMetric.values():
```

Comment:
~I kind of dislike this. @Keats what do you think about a `has_value` staticmethod or similar? That seems to be a suggestion here and is IMO a bit clearer than adding `values` in this same way.~

@clausmith @Keats already added this. I'd just lean towards writing as `AlphaDiversityMetric.has_value(metric)`